ios/PushNotificationPlugin/PushNotificationPlugin.m

```diff
@@ -5,14 +5,19 @@
 #import "UALocationService.h"
 #import "UAConfig.h"
 #import "NSJSONSerialization+UAAdditions.h"
+#import "SECommandQueue.h"
```

Comment:
I can extract it into a new repository and add it as a submodule if you think that's worth the effort :)